Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelConverters.hpp

```diff
@@ -21,6 +21,7 @@
 #include "Acts/Plugins/GeoModel/detail/GeoUnionDoubleTrdConverter.hpp"
 #include "Acts/Utilities/Result.hpp"
 
+#include <map>
```

Comment:
```suggestion
#include <unordered_map>
```

will require reformatting